app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -110,21 +110,16 @@ def extra_filters
       output
     end
 
-    # def current_work_ids
-    #   ActiveSupport::Deprication.warn('Bulkrax::CsvParser#current_work_ids will be replaced with #current_record_ids in version 3.0')
-    #   current_record_ids
-    # end
-
     def current_record_ids
       @work_ids = []
       @collection_ids = []
-      # @file_set_ids = []
+      @file_set_ids = []
```

Comment:
I had commented these out because I'm not fully sure what the goal of the `FileSet` is because for every `Work` that had a file, a `FileSet` was created.  In my test I had four works and therefore also four file sets.  These file sets were also getting bagged up but they are invalid, no data, RDF, CSV, nor checksums.  It's as if the `bag.manifest!` never executed, perhaps because of the lack of a file in the data directory?

I also tested what if a `Collection` was exported and it also gave me an invalid bag.

<img width="660" alt="image" src="https://user-images.githubusercontent.com/19597776/171664318-5300eba9-5d4f-471d-b177-c68a9dd277a8.png">

This then led me to question the one work per bag approach because I was thinking about the importing process of these works into a new tenant.  As far as I understand, the user would have then to import one bag at a time wouldn't they?  This seems to go against the concept of a bulk importer.  I am also fully cognizant of the fact that I'm completely misunderstanding the import process 😄...

Reply:
Thanks for pointing out these issues and concerns Kirk!